### PR TITLE
Update int_github__issue_joined.sql

### PR DESCRIPTION
**Pull request by jackiexsun**
Pull Request
**Are you a current Fivetran customer?** 
I work at Fivetran :)

**What change(s) does this PR introduce?** 
Fixes this https://github.com/fivetran/dbt_github/issues/30

**Did you update the CHANGELOG?** 

- [ ] Yes

How do I do that?
**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [ ] Yes

How do I do that?
**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature https://github.com/fivetran/dbt_github/issues/30
- [ ] No 

**How did you test the PR changes?** 

 

- [ ] CircleCi  
- [x] Local (please provide additional testing details below)

First i ran `dbt run -m +int_github__issue_joined` and then I ran `dbt run -m github__issues` and I examined the output.

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [ ] Redshift
- [ ] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🌮 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: models/intermediate/int_github__issue_joined.sql [hunk 1]
```diff
@@ -59,7 +59,7 @@ pull_request as (
 
 select
   issue.*,
-  {{ dbt_utils.concat(["'https://github.com/'",'repository_teams.repository',"'/pull/'", 'issue.issue_number']) }} as url_link,
+  {{ dbt_utils.concat(["'https://github.com/'",'repository_teams.repository',if(issue.pull_request,"'/pull/'", "'/issues/'"), 'issue.issue_number']) }} as url_link,
```
Comment (fivetran-joemarkiewicz) [inline models/intermediate/int_github__issue_joined.sql:62]: I wasn't able to get this sql to compile 🤔. It probably has something to do with the if statement within the jinja argument. I have seen these act up before.

It isn't as elegant, but I was able to get the code working with the suggested `case when` statement. Feel free to give it a test on your end and see if it works for you! I tested it on my side and saw the url be properly populated.
```suggestion
  case when issue.is_pull_request
    then {{ dbt_utils.concat(["'https://github.com/'",'repository_teams.repository',"'/pull/'", 'issue.issue_number']) }}
    else {{ dbt_utils.concat(["'https://github.com/'",'repository_teams.repository',"'/issues/'", 'issue.issue_number']) }}
      end as url_link,
```

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
Thanks so much for opening this PR @jackiexsun! 💯 I am so happy you are contributing to our packages!

I just tested your proposed changes locally and was having trouble with the query compiling properly. I added a suggestion in line that you can try on your end. It worked for the sample set I just used.

If you want to test these changes yourself, you can actually install your forked branch in place of the dbt hub package version! You can do that by removing the `fivetran/github` package dependency in the `packages.yml` file and using the below instead:
```yml
packages:
    - git: https://github.com/jackiexsun/dbt_github.git
      revision: patch-1
      warn-unpinned: false
```

This will allow you to install your forked version on `dbt deps` and run your code in place of the package code. Feel free to test this out and apply the suggestion I made. Let me know if you have any questions!

> Also, FYI since we are using dbt v0.20.2 internally we will use a similar git branch dependency style if you want to integrate these changes ASAP since the latest version of the package only supports dbt >= v1.0.0

**Comment by jackiexsun** [comment]
@fivetran-joemarkiewicz  thanks for the review! i converted it to your suggestion and ran the model locally, and it compiled and produced the right url links.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
Thanks again so much @jackiexsun for this contribution! The changes look great to me.

Also, I added a few updates to include the version update and also some CHANGELOG additions. All looks good to merge to me!

**Pull request merged**